packages/components/accordion/src/use-accordion-item.ts

```diff
@@ -0,0 +1,138 @@
+import type { PropGetter } from "@yamada-ui/core"
+import type { KeyboardEvent, KeyboardEventHandler } from "react"
+import type { AccordionItemOptions } from "./accordion-item"
+import { ariaAttr, handlerAll, isArray, mergeRefs } from "@yamada-ui/utils"
+import { useCallback, useId } from "react"
+import {
+  useAccordionContext,
+  useAccordionDescendant,
+} from "./accordion-context"
+
+interface useAccorDionItemProps
+  extends Pick<AccordionItemOptions, "isDisabled"> {}
+
+export function useAccorDionItem({ isDisabled }: useAccorDionItemProps) {
```

Comment:
@kaehehehe 
Here is a typo.Please correct it to `useAccordionItem`.

Reply:
Thank you for catching that!